File: C++/0210. 模除.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
  
    int a, b;
    while(cin>>a>>b) {
        while(a >= b) {
            a -= b;
        }
        cout<<a<<endl;
    }
    return 0;
}

File: Python/0210. 模除.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:  
            a, b = map(int, input().split())
            print(a % b)
        
        except EOFError:
            break
    
if __name__ == "__main__":
    func()
